### ISSUE TITLE
is not showing the preview layer

### ISSUE THREAD

**Issue by hugoerg56** [OpenKickflip/openkickflip-ios-sdk#3]
On the old kickflip sdk i use that on rubymotion and works really good, when i try to do this with openkicflip the preview layer is not showed and take a long time to load that view, any idea?:
```
  def on_load
    setupCameraView
  end  

  def on_init
    @recorder = KFRecorder.alloc.init
    #@recorder.s3Configuration = $s3Config
    @recorder.videoConnection.setVideoOrientation(AVCaptureVideoOrientationPortrait)
    @recorder.delegate = self
  end  

  def setupCameraView
    @preview = @recorder.previewLayer
    @preview.removeFromSuperlayer
    @preview.frame = CGRectMake(0, 0, Device.screen.width, Device.screen.height - 100)
    @preview.connection.setVideoOrientation(AVCaptureVideoOrientationPortrait)
    self.view.layer.addSublayer(@preview)
  end  
```

**Comment by duanmua** [comment]
@hugoerg56 I had the same issue. The problem is actually in KFRecorder, not KFBroadcastViewController. The problem lines were the following in setupSession():
```
    _movieWritingQueue = dispatch_queue_create("Movie Writing Queue", DISPATCH_QUEUE_SERIAL);
    _session.automaticallyConfiguresApplicationAudioSession = NO;
```
I should note though that I did not need the save to camera roll feature, which i believe these lines are a part of. Since that feature didn't matter to me, I took these 2 lines out and the previewLayer worked as expected again
